bears/java/CheckstyleBear.py

```diff
@@ -10,6 +10,13 @@
     "geosoft": "http://geosoft.no/development/geosoft_checks.xml"}
 
 
+def check_invalid_configuration(checkstyle_configs, use_spaces, indent_size):
+    if (checkstyle_configs is 'google' and
+            (not use_spaces or indent_size != 2)):
+        raise ValueError('Google checkstyle config does not support ' +
+                         'use_spaces=False or indent_size != 2')
```

Comment:
let's open another issue for that, at least we know from google that these settings don't fit together.